corehq/apps/app_manager/templates/app_manager/app_view_settings.html

```diff
@@ -18,6 +18,7 @@
   {% compress js %}
     <script src="{% static 'app_manager/js/settings/app_logos.js' %}"></script>
     <script src="{% static 'app_manager/js/settings/commcare_settings.js' %}"></script>
+    <script src="{% static 'app_manager/js/widgets.js' %}"></script><!-- version when copying app -->
```

Comment:
i thought these comments were only in JS files, we now add them in HTML as well?

Reply:
It's the same idea - noting why a dependency is there - whether it's in js or html.